### ISSUE TITLE
Isue when entering credentials

### ISSUE THREAD

**Issue by AmauryGouverneur** [hugodecasta/ZRMconnector#3]
Hi, 

I get this error message just after entering my one-time remarkable code. Any idea why? 

Thanks in advance, 

ReMarkable credentials Error
/Users/amaurygouverneur/MyDocuments/KTH/Research/zotero_remarkable_connector/ZRMconnector/node_modules/node-fetch/lib/index.js:683
		throw new TypeError(`${value} is not a legal HTTP header value`);
		      ^

TypeError: Bearer <!doctype html><html><head><meta charset="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=no,minimum-scale=1,maximum-scale=1"/><title>reMarkable</title><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':

**Comment by hugodecasta** [comment]
Yes the reMarkable API has changed since i uploaded this project (for security and reliability reasons mainly) and i haven't had time to update the project since.
So if you happen to have some time and are able to have some knowledge about the new RM API, give it a go ;)
